packages/ckeditor5-dev-utils/package.json

```diff
@@ -14,14 +14,14 @@
     "fs-extra": "^8.1.0",
     "javascript-stringify": "^1.6.0",
     "pofile": "^1.0.9",
-    "postcss": "^7.0.17",
-    "postcss-import": "^12.0.0",
-    "postcss-loader": "^3.0.0",
-    "postcss-mixins": "^6.2.0",
-    "postcss-nesting": "^7.0.0",
+    "postcss": "^8.2.12",
+    "postcss-import": "^14.0.1",
+    "postcss-loader": "^4.2.0",
+    "postcss-mixins": "^7.0.3",
```

Comment:
There seems to be a security issue with a module  dependent on  postcss-mixins.
I would appreciate it a lot if you could raise the priority of this.

--
@ckeditor/ckeditor5-dev-utils@25.4.2 requires glob-parent@^3.1.0 via a transitive dependency on fast-glob@2.2.7
[CVE-2020-28469](https://github.com/advisories/GHSA-ww39-953v-wcq6)